src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -559,14 +580,16 @@ const formConfig = {
         retirementPayWaiver: {
           title: 'Retirement pay waiver',
           path: 'retirement-pay-waiver',
-          depends: hasMilitaryRetiredPay,
+          depends: formData =>
+            hasMilitaryRetiredPay(formData) && !increaseOnly(formData),
           uiSchema: retirementPayWaiver.uiSchema,
           schema: retirementPayWaiver.schema,
         },
         trainingPayWaiver: {
           title: 'Training pay waiver',
           path: 'training-pay-waiver',
-          depends: formData => formData.hasTrainingPay,
+          depends: formData =>
+            formData.hasTrainingPay && !increaseOnly(formData),
```

Comment:
🤔 so in both flows we ask veterans if they have training pay, but in the increase flow we don't follow up on it? Kind of seems like we either should ask them about it and follow up or not ask them about it at all? https://github.com/department-of-veterans-affairs/vets-website/pull/9470/files#diff-e3260aa845f2b3d5d40e8ea9cea0e6d4L192

Reply:
Ditto with `retirementPayWaiver`, I think. There might be other places, I didn't check everything.

Reply:
Ooh, good eye! :male_detective:

@goldenmeanie I assume we should hide the pay questions at the front of the form for increase only since we're hiding the waiver pages here. Is that right?

@markgreenburg I'll look for anywhere else that may do this. Thanks!

Reply:
Yes, please hide for Increase-only.